### PR TITLE
CY-586 Enable log copy from remote to local when log saving is enabled

### PR DESCRIPTION
**Pull request by davidginzbourg**
- Setting `CFY_LOGS_PATH_REMOTE=SOME_PATH` will invoke the log saving at the end of each test (related code is in the `cloudify-manager` integration tests module).
- Setting `CFY_LOGS_PATH_LOCAL=SOME_PATH` will copy the logs from the containers to the local machine.
- Setting `SKIP_LOGS_EXTRACTION` will not remove the `logs.tar.gz` files and skip extraction.

### CODE REVIEW

**Comment by davidginzbourg** [comment]
Don't merge this one yet.

File: itests-runner/README.md [hunk 1]
```diff
@@ -252,7 +252,24 @@ For example:
 }
 ```
 
+### Saving the Cloudify Manager's logs
+Run 
 
+`export CFY_LOGS_PATH_LOCAL=YOUR_LOCAL_PATH && export CFY_LOGS_PATH=YOUR_REMOTE_PATH` 
+
+to save the logs
+to the local path you set.
```
Comment (tyacbovi) [inline itests-runner/README.md]: What about the remote path? is it necessary ?

File: itests-runner/README.md [hunk 2]
```diff
@@ -252,7 +252,24 @@ For example:
...
+You may want to run 
+
+`export DONT_REMOVE_LOG_TAR=True`
```
Comment (tyacbovi) [inline itests-runner/README.md]: See the manager review

File: itests-runner/README.md (hunk 1 above)
Comment (davidginzbourg) [reply itests-runner/README.md]: Yes, otherwise I don't know in advance which path to `scp` from.

File: itests-runner/README.md (hunk 2 above)
Comment (davidginzbourg) [reply itests-runner/README.md]: Changed.

**Pull request merged**